cmd/delete/delete_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package delete_test
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/cli-experimental/cmd/apply"
+	"sigs.k8s.io/cli-experimental/cmd/delete"
+	"sigs.k8s.io/cli-experimental/internal/pkg/wirecli/wirek8s"
+)
+
+var host string
+
+func setupKustomize(t *testing.T) string {
+	f, err := ioutil.TempDir("/tmp", "TestDelete")
+	assert.NoError(t, err)
+	err = ioutil.WriteFile(filepath.Join(f, "kustomization.yaml"), []byte(`apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
I found out this `Kustomization` manifest appears under `cmd/delete` and `cmd/prune`. Should we consider de-duplicating it under parent directory `cmd`?